### PR TITLE
Fix for issue 1099.

### PR DESCRIPTION
**Pull request by zanemcca**
Signed-off-by: zane mccaig <zanemccaig@gmail.com>

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by zanemcca** [comment]
fixes #1099

**Comment by zanemcca** [comment]
Could you review @raymondfeng?

File: common/models/role.js [hunk 1]
```diff
@@ -104,8 +104,11 @@ module.exports = function(Role) {
...
 
   function isUserClass(modelClass) {
-    return modelClass === loopback.User ||
-      modelClass.prototype instanceof loopback.User;
+    if (modelClass) {
+      return modelClass === loopback.User ||
+        modelClass.prototype instanceof loopback.User;
+    }
+    else return false;
   }
```
Comment (raymondfeng) [inline common/models/role.js]: Please make sure trailing spaces are removed. CI fails due to the style issue.

**Pull request merged**